Source/ThirdParty/gtest/xcode/Config/General.xcconfig

```diff
@@ -63,5 +63,7 @@ COMBINE_HIDPI_IMAGES = YES;
 // VERSIONING BUILD SETTINGS (used in Info.plist)
 GTEST_VERSIONINFO_ABOUT =  © 2008 Google Inc.
 
+PRODUCT_BUNDLE_IDENTIFIER = com.apple.$(PRODUCT_NAME:rfc1034identifier);
```

Comment:
FYI, I see that we have both `org.webkit.` and `com.apple.` bundle ID domains in tools code. There doesn't seem to be a rigid pattern, but it might be nice to use `org.webkit.` to mark projects that _aren't_ submitted to the OS. 
```suggestion
PRODUCT_BUNDLE_IDENTIFIER = org.webkit.$(PRODUCT_NAME:rfc1034identifier);
```